src/libraries/Microsoft.PowerFx.Interpreter/Environment/PowerFxConfigExtensions.cs

```diff
@@ -34,16 +34,31 @@ public static void EnableSetFunction(this PowerFxConfig powerFxConfig)
         /// Enable all multation functions which allows scripts to execute side effect behavior.
         /// </summary>
         /// <param name="symbolTable"></param>
+        [Obsolete("This will be removed. Enable mutation functions by calling PowerFxConfig.EnableMutationFunctions()")]
```

Comment:
> [Obsolete("This will be removed. Enable mutation functions by calling PowerFxConfig.EnableMutationFunctions()")] [](http://example.com/codeflow?start=7&length=113)

why obsolete? 
SymbolTable is better than Config - it's more granular. 

Config is global. 
SymbolTable can chain / share and be per formula.

Reply:
The challenge here was having CollectFcuntion in Core and its implementation in the Interpreter without adding Collect to the BuiltinCoreFunctions. We already have Set, Json and Regex functions being added to Config. Do we want only mutation functions at SymbolTable or are we planning on moving all these other functions to SymbolTable as well?